Expensify/App
# Fix text overflow in the report title hiding the pin icon

<Assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

<Explanation of the change or anything fishy that is going on>

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/144225 and https://github.com/Expensify/Expensify/issues/144237

### Tests
<Tests you performed on all platforms that validates your changed worked>

1. Update a user name or report name to be really really long 

1. Open that report chat or DM

1. Make sure you can still see the pin button


### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>

BEFORE
![Screen Shot 2020-10-22 at 11 15 36 AM](https://user-images.githubusercontent.com/30609178/96912989-ec0e4300-1457-11eb-9f8a-b93a29aa5d37.png)


AFTER
![Screen Shot 2020-10-22 at 11 15 16 AM](https://user-images.githubusercontent.com/30609178/96912926-ddc02700-1457-11eb-86ef-14ceb4a0eab3.png)
